guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -15,6 +15,29 @@ During migration to Pulp 3, the amount of data stored in the PostgreSQL data dir
 Ensure that you have enough space for the migration.
 A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
 
+.Pre-Migration Checks
+* Ensure that all enabled repositories complete the synchronization process.
+* If any repositories are synced with a *Warning* or *Error* state, fix the underlying problem and resync the repository.
+* Ensure that no paused tasks in the system are related to any repositories or Content Views.
+* If you no longer require specific Content Views or Composite Content View Versions, you can perform a cleanup action.
+* For more information about how to perform the cleanup, see https://access.redhat.com/solutions/2760531[How to remove old content view versions in Red Hat Satellite 6 using CLI/hammer?] .
+* If some Content View or Composite Content View versions are not required anymore, please consider performing a cleanup of them. https://access.redhat.com/solutions/2760531[How to remove old content view versions in Red Hat Satellite 6 using CLI/hammer?] can be referred to understand how to perform the cleanup of older Content View/Composite Content View versions using `hammer_cli`.
```

Comment:
Please maintain consistency with cross referencing. 
It should always be "For more information about x, see y"